src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -86,7 +101,7 @@ export const ChannelView: React.FC = () => {
         isPublic_eq: true,
       },
     },
-    { notifyOnNetworkStatusChange: true, fetchPolicy: 'cache-and-network' }
+    { notifyOnNetworkStatusChange: true }
```

Comment:
Why do we need `notifyOnNetworkStatusChange`?

Reply:
Needed because when `fetchingMore` we need the loading flag to show the placeholders without `notifyOnNetworkStatusChange` we don't get the loading flag